pycbc/live/snr_optimizer.py

```diff
@@ -401,4 +405,8 @@ def args_to_string(args):
         option_fullname = f'--snr-opt-{optimizer_name}-{opt}'
         argstr += f'{option_fullname} {option_value} '
 
+    # Add the extra opts
+    if args.snr_opt_extra_opts is not None:
+        argstr += args.snr_opt_extra_opts
+
```

Comment:
Yes this is what I suggested, but I am missing why you still need all the code above. Why does `--snr-opt-method` need special handling for example?

Reply:
It doesn't need any special handling if everything is passed through snr_opt_extra_opts 😁